src/util/random.js

```diff
@@ -0,0 +1,3 @@
+export default function () {
+	return Math.random().toString(16);
```

Comment:
we should use: `Math.random().toString(16).substr(2)` since we want a random hex string rather than '0.{hex}'.